tests/SideBySide/SchemaProviderTests.cs

```diff
@@ -49,7 +49,6 @@ public void ColumnsSchema()
 		AssertHasColumn("CHARACTER_MAXIMUM_LENGTH", typeof(long));
 		AssertHasColumn("NUMERIC_PRECISION", typeof(ulong));
 		AssertHasColumn("NUMERIC_SCALE", typeof(ulong));
-		AssertHasColumn("DATETIME_PRECISION", typeof(uint));
```

Comment:
This code exists to verify that changes haven't introduced a regression; you can't just delete it to make the tests pass.